### PR TITLE
Fix the ingest rate displayed in the CLI.

### PR DESCRIPTION
**Pull request by fulmicoton**
The CLI was showing the average rate since the CLI was started.

This PR changes it to an estimation over the last 2 seconds.

### CODE REVIEW

**Comment by trinity-1686a** [comment]
I tried it and it seems somehow broken. After a while, I get this:
```
⠋ [00:20:00] 20.82 GiB/20.82 GiB (91.4 MiB/s)
```
Ingestion rate should be in the order of 17MiB/s, but the number I got never seemed to go below 40MiB/s

**Comment by fulmicoton** [comment]
oh really? let me check!  Thanks for testing it!

**Comment by fulmicoton** [comment]
@trinity-1686a There was a bug in the rate estimator indeed. I added @guilload to the review.

**Comment by trinity-1686a** [comment]
the estimator always seems to give a multiple of 5.71 for this workload. This is because it essentially count batches of (quasi-)fixed size, and there aren't that many batches processed over a 2s window. In my case, there are almost always 3 batches processed, so I get 17.1MiB most of the time, and 11.4 or 22.8 when 2 or 4 batches got processed.
Maybe for that estimator, it would be better to count the time it took to send the last X batches, rather than to count how many batches were sent in X seconds.

File: quickwit/quickwit-common/src/tower/rate_estimator.rs [hunk 1]
```diff
@@ -252,36 +289,72 @@ mod tests {
...
+
+        // Now let's assume there isn't any work ongoing for 4s.
+        // Over the last 9 seconds, we have received 500 works
+        //
+        // After the reset, we should have the following buckets:
+        // We expect a mean of 44 work/s.
+        // |0, 0, 0, 0, 0, 100*, 100, 100, 100, 100|
+        //
+        // Since the current bucket (idx = 5) is not taken into account, this leads
+        // to an average of 500 / 9 = 55 work units.
```
Comment (trinity-1686a) [inline quickwit/quickwit-common/src/tower/rate_estimator.rs:357]:
```suggestion
        // to an average of 400 / 9 = 44 work units.
```

**Comment by fulmicoton** [comment]
I think there are different ways to improve the estimation, but I'll merge this for the bugfix. @trinity-1686a feel free to revisit it.

**Pull request merged**